assets/src/edit-story/constants.js

```diff
@@ -76,3 +76,8 @@ export const FONT_WEIGHT = {
   MEDIUM: 500,
   BOLD: 700,
 };
+
+export const HIDDEN_PADDING = {
+  horizontal: 6,
+  vertical: 2,
```

Comment:
I'm actually seeing `4` for vertical in the [issue description](https://github.com/google/web-stories-wp/issues/4972). Was this changed outside of the issue somewhere?

Reply:
I've seen both 6x4 & 6x2 in ticket descriptions. I'll verify which one sam wants

Reply:
Sam clarified and upddated the request to 8x4